sequencing/clinical_annotator/src/clinical_annotator.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+# clinical_annotator 1.0.0
+set -e -x -o pipefail
+
+#################################################
+# Downloads input VCF index files
+# Arguments: 1) VCF Index File 2) /home/dnanexus
+# Returns: None
+#################################################
+function parallel_download() {
+	cd $2
+	dx download "$1"
+	cd - >/dev/null
+}
+export -f parallel_download
+
+###########################################
+# Downloads and reformats resource files
+# Arguments: None
+# Returns: None
+############################################
+function reformat_resources() {
+
+	cd $HOME
+
+	#python $HOME/ClinVar_tsvTOvcf.py variant_summary.txt.gz
+	python ClinVar_tsvTOvcf.py variant_summary.txt.gz
+    
+	vcf-sort -c variant_summary.b37.vcf | bgzip -c > variant_summary.b37.vcf.gz
```

Comment:
Should it always do files for both build versions, or should it look at the $build_version variable and pick one?